notebooks/official/experiments/build_model_experimentation_lineage_with_prebuild_code.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>reduce to:</p><p>Install additional package dependencies not installed in your notebook environment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>